### PR TITLE
The game works. Added account display

### PR DESCRIPTION
**Pull request by VikTorNik**
### Frogger Game

[Demo](https://viktornik.github.io/frontend-nanodegree-arcade-game/) |
[Code base](https://github.com/VikTorNik/frontend-nanodegree-arcade-game)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by VikTorNik** [comment]
Hey!
I fixed bugs Const for Fogger.
Please, review.

File: submissions/VikTorNik/froggen-oojs/app.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        i = new Enemy(-GAME_CONF.OffsetX, GAME_CONF.OffsetY * Math.floor(Math.random() * 3 + 1) - ENEMY_CONF.startY, ENEMY_CONF.speed));
+};
+
+createEnems();
+
+Enemy.prototype.collision = function () {
+    ((player.x < this.x + GAME_CONF.OffsetX * GAME_CONF.overlap) &&
+        (player.x > this.x - GAME_CONF.OffsetX * GAME_CONF.overlap) &&
+        (player.y < this.y + GAME_CONF.OffsetY * GAME_CONF.overlap) &&
+        (player.y > this.y - GAME_CONF.OffsetY * GAME_CONF.overlap)) ? (player.x = PLAYER_CONF.startX, player.y = PLAYER_CONF.startY, createEnems(), GAME_CONF.countСheckmate += 1) : false;
```
Comment (OleksiyRudenko) [inline submissions/VikTorNik/froggen-oojs/app.js:52]: See item 4.iii of the requirements.

File: submissions/VikTorNik/froggen-oojs/app.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+};
+
+Enemy.prototype.winner = function () {
+    player.y < 0 ? (player.x = PLAYER_CONF.startX, player.y = PLAYER_CONF.startY, createEnems(), GAME_CONF.countWinner += 1) : false;
```
Comment (OleksiyRudenko) [inline submissions/VikTorNik/froggen-oojs/app.js:56]: Is that an enemy can be a winner?
Why in Enemy class context?

File: submissions/VikTorNik/froggen-oojs/app.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+Enemy.prototype.render = function () {
+    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+};
+
+function createEnems() {
+    allEnemies = [...Array(Math.floor((3 + Math.random() * (ENEMY_CONF.maximumCount - 2))))].map((_, i) =>
+        i = new Enemy(-GAME_CONF.OffsetX, GAME_CONF.OffsetY * Math.floor(Math.random() * 3 + 1) - ENEMY_CONF.startY, ENEMY_CONF.speed));
+};
+
+createEnems();
```
Comment (OleksiyRudenko) [inline submissions/VikTorNik/froggen-oojs/app.js:46]: Do not intermix class definitions with other code. 
Definitions, then executions.
Tidy your code up.

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@VikTorNik good start. Needs improvement.
Double-check yourself against the [requirements](https://kottans.org/documentation/docs/doc/code-review/#oo-js-frogger)

**Comment by VikTorNik** [comment]
Hey!
I fixed bugs.
Please, review.

File: submissions/VikTorNik/froggen-oojs/app.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+Player.prototype.render = function () {
+    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+    this.countWinnerСheckmate();
+};
+
+Player.prototype.winner = function () {
+    this.y < 0
+        ? (this.x = PLAYER_CONF.startX, this.y = PLAYER_CONF.startY, GAME_CONF.countWinner += 1)
+        : false;
```
Comment (OleksiyRudenko) [inline submissions/VikTorNik/froggen-oojs/app.js:82]: Please explain this. The result of ternary is never used. What is the purpose?

File: submissions/VikTorNik/froggen-oojs/app.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    this.offsetMaxY = PLAYER_CONF.offsetMaxY;
+    this.winner();
+};
+
+Player.prototype.render = function () {
+    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+    this.countWinnerСheckmate();
+};
+
+Player.prototype.winner = function () {
```
Comment (OleksiyRudenko) [inline submissions/VikTorNik/froggen-oojs/app.js:79]: By convention, function names start with a verb to denote action.
What this function is intended to **do**?

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@VikTorNik please fix unobvious code fragments

File: submissions/VikTorNik/froggen-oojs/app.js (hunk 4 above)
Comment (VikTorNik) [reply submissions/VikTorNik/froggen-oojs/app.js:82]: Using the ternary operator is more concise and more concise. I like this option better than using "if".

```
    if (this.y < 0) {
        this.x = PLAYER_CONF.startX;
        this.y = PLAYER_CONF.startY;
        GAME_CONF.countWinner += 1;
    }
```

File: submissions/VikTorNik/froggen-oojs/app.js (hunk 5 above)
Comment (VikTorNik) [reply submissions/VikTorNik/froggen-oojs/app.js:79]: You're right. I have corrected the name of the function.

**Comment by VikTorNik** [comment]
I fixed bugs.
Please, review.

File: submissions/VikTorNik/froggen-oojs/app.js (hunk 4 above)
Comment (OleksiyRudenko) [reply submissions/VikTorNik/froggen-oojs/app.js]: We write code for other people. Computers are ok with byte-codes.
The purpose of a ternary operator is to provide a value. `false` that is never used creates feeling of incompleteness.

**Review by OleksiyRudenko — APPROVED** [review]
@VikTorNik well done!

**Pull request merged**